tests/examples/shims/screen.lua

```diff
@@ -24,8 +24,18 @@ local function create_screen(args)
         geo.height = args2.height or geo.height
     end
 
+    s.outputs = {{
+        mm_width  = 0,
+        mm_height = 0,
+    }}
```

Comment:
The key of the table should be the name of the output, e.g. `LVDS1`, `VGA-1` or things like that. Dunno if it really matters here, but with that argument it would also be okay just not to add this at all...?

Reply:
It is currently not used by `awful` or any of my tests, so it doesn't need to be in the shims (yet).